google-oauth-client-appengine/pom.xml

```diff
@@ -30,12 +30,12 @@
             <manifest>
               <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
             </manifest>
+            <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
           </archive>
         </configuration>
         <executions>
```

Comment:
You can drop these `maven-jar-plugin` `<executions>` specs entirely since they just duplicate what's already specified by the default lifecycle.